### PR TITLE
Send content_id to content store and panopticon

### PR DESCRIPTION
**Pull request by tijmenb**
All taggable content on GOV.UK needs a content_id. This PR adds the content_id to the data being sent to panopticon and adds a rake task to send the item to the publishing-api.

Trello: https://trello.com/c/ZqWkgsej

### CODE REVIEW

**Comment by bishboria** [comment]
:+1:

**Comment by bishboria** [comment]
(Leaving the PR open for a bit longer, in case others want to review)

File: spec/notifiers/publishing_api_notifier_spec.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+RSpec.describe PublishingApiNotifier do
+  describe '#publish' do
+    it 'publishes the content item' do
+      allow(Services.publishing_api).to receive(:put_content_item)
+
+      PublishingApiNotifier.new.publish
+
+      expect(Services.publishing_api).to have_received(:put_content_item).with(
+        "/business-finance-support-finder",
+        be_valid_against_schema('placeholder')
```
Comment (mobaig) [inline spec/notifiers/publishing_api_notifier_spec.rb]: As discussed, this could be re-worked to be clearer.

**Comment by KushalP** [comment]
Are all reviewers done (and happy) with this PR?

**Comment by mobaig** [comment]
LGTM

**Pull request merged**